docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,9 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+0.12.23 - 2019-06-05
```

Comment:
It would be advisable to add a stable anchor. The last one if for version 0.12.18 and it's defined as follows:

```rst
.. _release-0-12-18:
```